CONTRIBUTING.md

```diff
@@ -0,0 +1,111 @@
+- The Cluster-Version-Operator(CVO) manages the lifecycle of the marketplace-operator along with other internal Openshift components. Before deleting the marketplace-operator deployment, instruct CVO to stop managing the marketplace-operator 
+```
+$ oc apply -f examples/cvo.override.yaml
+``` 
+- Delete the marketplace-operator deployment
+```
+$ oc delete deployment openshift-marketplace -n marketplace-operator
+```
+- Install [Operator-SDK](https://github.com/operator-framework/operator-sdk).
+- Compile the marketplace-operator and start the operator in your dev environment
+```
+$ operator-sdk up local --namespace=openshift-marketplace --kubeconfig=<path-to-kubeconfig-file>  
+```
+
+### Testing changes to RBAC policy
+- If there's any modification in existing RBAC policies, the modifications need to be tested using a separate deployment. Easiest way to do this is to build an image locally using [this dockerfile](./Dockerfile), push the image to a registry, and replace the [image here](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/operator.yaml#L23) with the newly built image. The operator can then be redeployed with the new changes.   
+
+```
+$ cd <path-to-operator-marketplace-repo>
+
+$ operator-sdk build . --tag=<REGISTRYHOST>/<USERNAME>/marketplace-operator
+
+$ docker push <REGISTRYHOST>/<USERNAME>/marketplace-operator 
+
+$ sed -i -e 's?quay.io/openshift/origin-operator-marketplace:latest?<REGISTRYHOST>/<USERNAME>/marketplace-operator?g' deploy/operator.yaml
+
+$ oc apply -f deploy/operator.yaml
+```
+
+## Marketplace-operator on vanilla Kubernetes
+
+- To set up marketplace-operator on vanilla Kubernetes, [install Operator-Lifecycle-Manager(OLM)](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/install/install.md) in a Kubernetes Cluster. 
+- `cd` into the marketplace-operator repository and install the marketplace-operator using 
+```
+$ kubectl apply -f deploy/upstream
+```
+- To delete the marketplace-operator deployment use 
+```
+$ oc delete deployment marketplace
+``` 
+
+### Testing changes to RBAC policy
+- If there's any modification in existing RBAC policies, the modifications need to be tested using a separate deployment. Easiest way to do this is to build an image locally using [this dockerfile](./Dockerfile), push the image to a registry, and replace the [image here](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/upstream/07_operator.yaml#L28) with the newly built image. The operator can then be redeployed with the new changes.
+
+```
+$ cd <path-to-operator-marketplace-repo>
+
+$ operator-sdk build . --tag=<REGISTRYHOST>/<USERNAME>/marketplace-operator
+
+$ docker push <REGISTRYHOST>/<USERNAME>/marketplace-operator 
+
+$ sed -i -e 's?quay.io/openshift/origin-operator-marketplace:latest?<REGISTRYHOST>/<USERNAME>/marketplace-operator?g' deploy/upstream/07_operator.yaml
+
+$ oc apply -f deploy/upstream/operator.yaml
```

Comment:
`kubectl apply`

Reply:
@SamiSousa thank you for noticing this. Addressed it.